crates/evm/src/evm/system_contracts/mod.rs

```diff
@@ -84,29 +84,22 @@ impl Bridge {
         // This is equal to:
         //
         // BridgeContract::initializeCall {
-        //     _depositScript: hex!("d2205daf577048c5e5a9a75d0a924ed03e226c3304f4a2f01c65ca1dab73522e6b8bad206228eba653cf1819bcfc1bc858630e5ae373eec1a9924322a5fe8445c5e76027ad201521d65f64be3f71b71ca462220f13c77b251027f6ca443a483353a96fbce222ad200fabeed269694ee83d9b3343a571202e68af65d05feda61dbed0c4bdb256a6eaad2000326d6f721c03dc5f1d8817d8f8ee890a95a2eeda0d4d9a01b1cc9b7b1b724dac00630663697472656114").into(),
-        //     _scriptSuffix: hex!("0800000000000f424068").into(),
-        //     _requiredSigsCount: U256::from(5),
+        //     _scriptPrefix: hex!("4a20a9dc2892380bf8245eb7a677ab166c98514ea6c5ce7946adbf4c05afb2bfbffdac00630663697472656114").into(),
+        //     _scriptSuffix: hex!("08000000003b9aca0068").into(),
+        //     _depositAmount: U256::from(10 ether),
```

Comment:
What is ether in the context?

Reply:
ether -- it's a unit in evm 

mean x 10 ** 18

Reply:
I can write directly 10 * 10 ** 18 if you want, I didn't do it as it's a comment